examples/list-files-and-folders/.devcontainer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "image": "ballerina/ballerina-devcontainer:2201.10.2",
+  "customizations": {
+    "vscode": {
+      "extensions": ["WSO2.ballerina"]
+    }
+  }
+}
```

Comment:
No need to commit this file

Reply:
Removed this file from both the examples.